tests/test_transcript.py

```diff
@@ -232,76 +257,40 @@ def test_commands_at_invocation():
         out = app.stdout.buffer
         assert out == expected
 
-
-def test_transcript_from_cmdloop(request, capsys):
-    # Create a cmd2.Cmd() instance and make sure basic settings are like we want for test
-    app = CmdLineApp()
-    app.feedback_to_output = True
-
-    # Get location of the transcript
-    test_dir = os.path.dirname(request.module.__file__)
-    transcript_file = os.path.join(test_dir, 'transcript.txt')
-
-    # Need to patch sys.argv so cmd2 doesn't think it was called with arguments equal to the py.test args
-    testargs = ['prog', '-t', transcript_file]
-    with mock.patch.object(sys, 'argv', testargs):
-        # Run the command loop
-        app.cmdloop()
-
-    # Check for the unittest "OK" condition for the 1 test which ran
-    expected_start = ".\n----------------------------------------------------------------------\nRan 1 test in"
-    expected_end = "s\n\nOK\n"
-    out, err = capsys.readouterr()
-    if six.PY3:
-        assert err.startswith(expected_start)
-        assert err.endswith(expected_end)
-    else:
-        assert err == ''
-        assert out == ''
-
-
-def test_multiline_command_transcript_with_comments_at_beginning(request, capsys):
-    # Create a cmd2.Cmd() instance and make sure basic settings are like we want for test
-    app = CmdLineApp()
-
-    # Get location of the transcript
-    test_dir = os.path.dirname(request.module.__file__)
-    transcript_file = os.path.join(test_dir, 'multiline_transcript.txt')
-
-    # Need to patch sys.argv so cmd2 doesn't think it was called with arguments equal to the py.test args
-    testargs = ['prog', '-t', transcript_file]
-    with mock.patch.object(sys, 'argv', testargs):
-        # Run the command loop
-        app.cmdloop()
-
-    # Check for the unittest "OK" condition for the 1 test which ran
-    expected_start = ".\n----------------------------------------------------------------------\nRan 1 test in"
-    expected_end = "s\n\nOK\n"
-    out, err = capsys.readouterr()
-    if six.PY3:
-        assert err.startswith(expected_start)
-        assert err.endswith(expected_end)
-    else:
-        assert err == ''
-        assert out == ''
-
-
 def test_invalid_syntax(_cmdline_app, capsys):
     run_cmd(_cmdline_app, 'speak "')
     out, err = capsys.readouterr()
     expected = normalize("""ERROR: Invalid syntax: No closing quotation""")
     assert normalize(str(err)) == expected
 
 
-def test_regex_transcript(request, capsys):
-    # Create a cmd2.Cmd() instance and make sure basic settings are like we want for test
+@pytest.mark.parametrize('filename, feedback_to_output', [
```

Comment:
This was a smart way to expand the test coverage for transcripts essentially just by creating the transcripts themselves.  I'm jealous I didn't think of it first ;-)